### DISCUSSION TITLE
How to set `raiseErrors` to `true`

### DISCUSSION THREAD

**Discussion by davidabou** [adonisjs/core#4650] [Help]
I want to use 
```ts
await response.download('image.png')
```

But to make this the help popup says :
```
If raiseErrors=false, then this method will self handle all the exceptions by writing a generic HTTP response. To have more control over the error, it is recommended to set raiseErrors=true and wrap this function inside a try/catch statement.
```

How can I set raiseErrors to true ?

**Comment by bosunski** [comment]
Will be nice to know how exactly this can be done.

**Comment by thetutlage** [comment]
Its covered here in the docs. https://docs.adonisjs.com/guides/basics/response#downloading-files

You can self handle the error by specifying a callback
